frb/figures/finder.py

```diff
@@ -5,16 +5,43 @@
 from IPython import embed
 
 from matplotlib import pyplot as plt
+from matplotlib import font_manager
+import matplotlib.cm as cm
+
+from PIL import Image
 
 from astropy import units
 from astropy.visualization.wcsaxes import SphericalCircle
 from astropy.stats import sigma_clipped_stats
 from astropy.visualization import LogStretch, mpl_normalize as mplnorm
 from astropy.nddata import Cutout2D
+from astropy.wcs import WCS
+from astropy.coordinates import ICRS
+from astropy.time import Time
 
 from linetools import utils as ltu
 
 from frb.figures import utils
+from frb.surveys import survey_utils
+from frb.surveys import images
+
+
+def from_hdu(hdu, title, **kwargs):
+    """
+    Convenience function to handle an HDU to generate a finder chart
+
+    Args:
+        hdul (astropy.io.fits.PrimaryHDU):
+        title (str):
+
+    Returns:
+        see generate()
```

Comment:
Can you copy the documentation of generate() here as well? At least the object returned, with a brief explanation: matplotlib.pyplot.figure, matplotlib.pyplot.Axis

Reply:
done